gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Tokens.java

```diff
@@ -60,4 +60,18 @@ private Tokens() {}
 
     public static final String STATUS_ATTRIBUTE_EXCEPTIONS = "exceptions";
     public static final String STATUS_ATTRIBUTE_STACK_TRACE = "stackTrace";
+    /**
+     * A {@link ResultSet#statusAttributes()} key for user-facing warnings.
+     * <p>
+     * The console checks for this key on remote results.  If the key is present,
```

Comment:
I think it would be good to generalize this javadoc a bit to not mention the "console", since any application could essentially take advantage of the warnings that are found here.

Reply:
Sounds good, I'll try to reword it.

Reply:
I've generalized it to talk about "recommended values" rather than console behavior.  The prose describing how things are passed through `String.valueOf` is still especially convoluted.  

This is how it reads now:

```
* A `java.util.List` implementation containing references for which `String.valueOf(Object)` produces a meaningful return value. For example, a list of strings.
* Otherwise, any single non-list object for which `String.valueOf(Object)` produces a meaningful return value. For example, a string.
```

But I could change it to just this:

```
* A list of strings
* A string
```

It's not technically as accurate -- the only client consumer is the console, and it currently does pass stuff through `String.valueOf` -- but it's safe, since those two bullets describe a subset of what's actually consumable.  It's also a lot more readable.

Please let me know if you want me to take another stab at fixing that javadoc, or if you have your own edit in mind, feel free to push it on there.  I recognize it's important to get any implementations that might use this on the same page as early as possible.